department-of-veterans-affairs/abd-vro
# added migration script and code changes for action and actionResultNa…

## What was the problem?


Associated tickets or Slack threads:
- #2049

## How does this fix it?[^1]


## How to test this PR
- run e2e sve-bie-kafka integration test


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
